api/v1/loadtest_types.go

```diff
@@ -265,6 +319,14 @@ type Client struct {
 	// +optional
 	Build *Build `json:"build,omitempty"`
 
+	// Sidecar describes the Sidecar container, which is the runtime of
+	// the proxy for the test associated with a proxy.
+	Sidecar *Sidecar `json:"sidecar,omitempty"`
```

Comment:
This is a great point, I always thought we need to wrap the container with something, but seems like all the required fields are in the [corev1.Container](https://github.com/kubernetes/api/blob/v0.20.2/core/v1/types.go#L2189). Hi @paulosjca, do you remember why we need to wrap the clone, build and run containers instead of use the container directly?

Reply:
As I discussed offline with @paulosjca, we may change the loadTest configuration definition to be more generic (make all run) before this #255, and the xdsServer and sidecar maybe just Containers.